### PR TITLE
Update crates for compatibility

### PR DESCRIPTION
**Pull request by dkarama**
Fix for https://github.com/ArniDagur/python-adblock/issues/36. All tests pass. Tested with qutebrowser-2.1.1 on OpenBSD-current

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -2,7 +2,6 @@
 #![deny(
     future_incompatible,
     nonstandard_style,
-    rust_2018_idioms,
```
Comment (ArniDagur) [inline src/lib.rs:5]: Why did you need to remove this?

**Comment by ArniDagur** [comment]
I did an update in https://github.com/ArniDagur/python-adblock/pull/42. However, thanks for your contribution.

File: src/lib.rs (hunk 1 above)
Comment (The-Compiler) [reply src/lib.rs:5]: https://github.com/PyO3/maturin/pull/496#issuecomment-817338064 has some context

Comment (ArniDagur) [reply src/lib.rs:5]: All right, thanks. It's an issue that only occurs when I use the git version of `PyO3`. It's something I'll need to deal with when `0.14` is released.
